CommunityProject/Assets/_Game/Scripts/Runtime/Settings/SettingsSystem.cs

```diff
@@ -73,9 +89,9 @@ private void ApplyGraphicSettings()
 			QualitySettings.SetQualityLevel(Settings.Graphic.GraphicLevel);
 		}
 
-		private float GetNormalizedToMixerVolume(float value)
+		private void ApplyLocalizationSettings()
 		{
-			return (value - 1) * 80;
+			LocalizationSettings.SelectedLocale.Identifier = Settings.Localization.Locale;
```

Comment:
```suggestion
			LocalizationSettings.SelectedLocale = LocalizationSettings.AvailableLocales.GetLocale(Settings.Localization.Locale);
```

Ich fürchte, dass Dein Code aktuell nur von der aktuellen Locale den Identifier ändert. 🙈